### DISCUSSION TITLE
wish language perl

### DISCUSSION THREAD

**Discussion by Laurent-Bardi** [wasmerio/wasmer#5266] [Ideas]
Thanks for proposing a new feature!

### Motivation

I use it through apache-asp-perl, but i think it would be more "open" with wasmer

### Additional context

I know that per seems "old" and you can do better with python, but, PCRE is used all around the world an P is not for python :-}

**Comment by xdoardo** [comment]
Thank you! I think we won't be able to allocate the right amount of time to do it. However, we always welcome contributions. The way to achieve this goal would imply compiling perl to [wasix](https://github.com/wasix-org).

**Reply by Laurent-Bardi** [reply]
I've followed the link, but i do not understand all. Could you explain me more about the process, i could give a try if i can.
